### PR TITLE
Towards mono repo

### PR DESCRIPTION
**Pull request by hemarkus**
- merge back proto and python stuff
- reorganize repo structure to reflect components not programming languages
- introduce ONE versioning file
- migrate from dep to go modules

### CODE REVIEW

File: versions.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+GRPCIO_VERSION=1.15.0
+DGRAPH_VERSION=1.0.9
+HUGO_VERSION=0.39
+SCANCODE_VERSION=2.9.1
+GO_VERSION=1.12.4
```
Comment (mirkoboehm) [inline versions.env]: Missing newline at end of file.

**Comment by hemarkus** [comment]
retest this please

File: doc/Makefile.common [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	@rm -fr $(HUGOTMP)/hugo
+
+doc/documentation: $(HUGO)
+	@cd doc && $(HUGO) -d documentation
+
+doc/qmstr-doc.tar.bz2: doc/documentation
+	@cd doc && tar -cjvf qmstr-doc.tar.bz2 documentation
+
+cleandocs:
+	@rm -rf doc/documentation doc/qmstr-doc.tar.bz2
```
Comment (mirkoboehm) [inline doc/Makefile.common]: Missing newline at end of file.

**Comment by hemarkus** [comment]
retest this please

**Pull request merged**